tools/InstallationTests/NetcoreTests/Resources/PolicyTests.ps1

```diff
@@ -24,12 +24,14 @@ function Test-PolicyDefinitionCRUD
 
 	# Test
 	$actual = New-AzureRMPolicyDefinition -Name $policyName -Policy SamplePolicyDefinition.json
+	Start-Sleep 5
```

Comment:
We shopuld actually have a test/retry loop here, as it may take more than 5 seconds.

Reply:
I'm hedging for xUnit support mid November. :)

Reply:
Good luck with that.  Meanwhile, there is an actual common function that does retries.  Please use it here

Reply:
Here is the retry function: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L338